orchestration/orchestrate_unified.py

```
@@ -67,77 +58,140 @@ def _check_dependencies(self):
+        """Find recent agent work that matches the task context."""
+        try:
+            # Check recent PRs for agent work
+            result = subprocess.run(
+                ["gh", "pr", "list", "--author", "@me", "--limit", "5", "--json", "number,title,headRefName,createdAt"],
+                capture_output=True, text=True, check=True
+            )
+            prs = json.loads(result.stdout)
+
+            # Look for recent agent PRs (created in last hour)
+            from datetime import datetime, timedelta
+            one_hour_ago = datetime.now() - timedelta(hours=1)
```

Ruff (0.12.2)

Local variable `one_hour_ago` is assigned to but never used

Remove assignment to unused variable `one_hour_ago`

(F841)